pylint_odoo/test/main.py

```diff
@@ -432,6 +433,42 @@ def test_140_check_migrations_is_not_odoo_module(self):
         expected_errors = {}
         self.assertDictEqual(real_errors, expected_errors)
 
+    @unittest.skipUnless(
+        sys.version_info[0:2] >= (3, 6),
```

Comment:
I think you even don't need index range here [0:2]

```suggestion
        sys.version_info >= (3, 6),
```

Reply:
True, I'll remove it.